src/pages/MenuPage.js

```diff
@@ -38,96 +95,66 @@ export class MenuPage extends React.Component {
     database.removeListener(this.databaseListenerId);
   }
 
+  /**
+   * Method will try to match stringified customData object for current store
+   * against keys in MODULES, will return values of keys in MODULES
+   * that match: customData[keyInModules]data === 'true'
+   * @param {string}  customDataAsString  Store custom data as string
+   * @return {array}  result              The maxOrdersPerPeriod of the orderType
+   */
+  getUsedModules = customDataAsString => {
+    if (!customDataAsString || customDataAsString.length === 0) return [];
```

Comment:
`''` is falsey already

Reply:
what if it's null/undefined though ? using it here to safely check length

Reply:
I mean `!customDataAsString` covers all three but this is more expliciti

Reply:
ohh true, i completely missed that one, the only way this would fail if customDataAsString is a true boolean or a number, which should never be the case, fixed: https://github.com/openmsupply/mobile/pull/868/commits/ece8f366218fe2d66debe189f0ed3247d0564a1b